packages/bot-web-ui/src/components/dashboard/tutorial-tab/common/search-input/index.tsx

```diff
@@ -15,7 +15,7 @@ const SearchInput = observer(({ faq_value, setFaqSearchContent, prev_active_tuto
         debounce(
             value => {
                 filterTuotrialTab(value);
-                setActiveTabTutorial(2);
+                setActiveTabTutorial(3);
```

Comment:
maybe it would be better to create a config with some explanatory names instead of using magic numbers, wouldn't it?
as an example 
```
export const DBOT_TABS: TDashboardTabsProps = Object.freeze({
    DASHBOARD: 0,
    BOT_BUILDER: 1,
    CHART: 2,
    TUTORIAL: 3,
});
```
usage example:
`const { BOT_BUILDER, CHART } = DBOT_TABS;`

Reply:
@maryia-matskevich-deriv sure we could do that. But i think it would be better to do it as an improvement because its not part of this PR's scope and it requires to refactor and make changes to all the other files that are related the tutorial tab. We'll do it in another PR and move this PR forward 👍